packages/loading-screen/src/main.js

```diff
@@ -4,17 +4,10 @@ import { BootstrapVue, IconsPlugin } from 'bootstrap-vue';
 import EkComponents from 'ek-components';
```

Comment:
Is `EkComponents` even needed after this? It looks to me like the only thing used from `EkComponents` is `loading-animation.gif`. It doesn't seem like any of the actual components from `EkComponents` are being used. Which means that the `kolibi.utils.i18n` issue can be sidestepped for now.

Reply:
If all that's needed is the gif, then maybe it could be duplicated and the ek-components dependency could be dropped (which would also make the standalone loading-screen smaller). Or you could move the gif into loading-screen and make loading-screen a dependency of the frontend.

Reply:
Ah true! If only that GIF asset is left then let's copy it and remove the dependency. I will append a new commiit.